invenio_accounts/models.py

```diff
@@ -72,6 +76,22 @@ def __str__(self):
     """List of fields that can be updated with update_profile."""
 
     @staticmethod
+    def get_by_nickname(nickname):
+        """Get a user by their nickname."""
+        try:
```

Comment:
Agree. I do think it makes sense to have query methods like ``get_by_nickname`` by I would either simply raise ``NoResultFound`` or return ``None``.